ballerina/root_logger.bal

```diff
@@ -147,6 +147,25 @@ isolated class RootLogger {
                 logRecord[k] = v;
             }
         }
+        if observe:isObservabilityEnabled() {
+            string? runtimeId = observe:getTagValue("runtime");
+            if runtimeId is string {
+                logRecord["runtime"] = runtimeId;
```

Comment:
I am not sure whether overriding the user defined values(with the same keys) here is a correct approach or not?
Ideally if we are expecting these keys to be added by our runtime, we should not allow them for user to configure

How about having one uniquely named field with a map value? (Same applies to tracing information)

@daneshk / @ThisaruGuruge WDYT?